package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eyeglass",
   "description": "Sass modules for npm.",
-  "version": "0.6.5",
+  "version": "0.6.6",
```

Comment:
it was an accident while i was trying to figure out the grunt/gulp tasks. gonna remove it.